README.md

```diff
@@ -10,7 +10,7 @@ System (GFS), GFS Ensemble Forecast System (GEFS), North American
 Mesoscale (NAM), Rapid Refresh (RAP), High Resolution Rapid Refresh
 (HRRR), Short Range Ensemble Forecast (SREF), Hurricane WRF (HWRF)
 applications, and is also used in Unified Forecasting System (UFS)
-applications.
+applications (MRW, SRW, and HAFS).
```

Comment:
@fossell RRFS might be added, or it is included in SRW?

Reply:
@WenMeng-NOAA - It's a configuration option within SRW. I didn't state RRFS explicitly here since I didn't think it was fully supported on its own yet (outside of using it with SRW) for the community. But I can include it if you think we should and would be helpful for users to know it's possible. Or we can add later when it becomes operational and RAP/HRRR are retired?  Let me know your preference.

Reply:
Per upcoming NCEP implementation perspective, it might be good to mention RRFS since the parallel run is on-going.

Reply:
Sounds good, I updated that section a little bit more. Let me know what you think.

Reply:
@fossell Looks good to me. Thanks!